### PR TITLE
Add mediahuman-audio-converter again

### PR DESCRIPTION
**Pull request by jnooree**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### Additional Contexts

Although this cask was removed in #118246, this app is still being developed (looks like). It has been updated to 1.9.8.1 on Jan 2022.

### CODE REVIEW

**Comment by bevanjkay** [comment]
@jnooree Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

File: Casks/mediahuman-audio-converter.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  url "https://www.mediahuman.com/download/MHAudioConverter.dmg"
+  name "MediaHuman Audio Converter"
+  desc "Audio converter"
+  homepage "https://www.mediahuman.com/audio-converter/"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
+    regex(/"softwareVersion">(\d+(?:\.\d+)*)/i)
```
Comment (neutric) [inline Casks/mediahuman-audio-converter.rb:13]:
```suggestion
    regex(/"softwareVersion">\s*(\d+(?:\.\d+)+)[\s<]/i)
```

**Comment by jnooree** [comment]
I actually ran `brew createzap` before opening this, but it reported nothing. I'm pretty busy now, so maybe I could take a look into the common directories later this week.

**Comment by neutric** [comment]
```
% brew createzap mediahuman
  zap trash: [
    "~/Library/Preferences/com.mediahuman.Audio Converter.plist",
    "~/Library/Preferences/com.mediahuman.plist",
    "~/Library/Saved Application State/com.mediahuman.Audio Converter.savedState",
  ]
```
You probably didn't use the right search term and/or didn't start the app once prior to running `brew createzap` (so that it can create its files).

**Comment by jnooree** [comment]
Thank you for comments! Added in jnooree@7d77374.

**Comment by bevanjkay** [comment]
Thank you @jnooree

**Pull request merged**